### PR TITLE
Added Ionic 2 Package

### PR DESCRIPTION
**Pull request by Bernardoow**
Please provide the following information:

 - https://github.com/Bernardoow/Ionic2Package
 - https://github.com/Bernardoow/Ionic2Package/tags 

This package is for new Ionic 2. New html tags and I will adding auto-completion and others things.

@FichteFoll 
Maybe don't remember, but I learned with you. Heheh For This package I created a branch to do pr hehehe.

### CODE REVIEW

File: repository/i.json [hunk 1]
```diff
@@ -810,6 +810,17 @@
 			]
 		},
 		{
+			"name": "Ionic 2 Package",
```
Comment (wbond) [inline repository/i.json]: This should be changed to something like `Ionic 2 Snippets`. The word `Package` should definitely not be there.

Comment (Bernardoow) [inline repository/i.json]: Hi @wbond ,

I will improve this package with more things. Like completions. Because this I put this name with ionic 2 package.

Comment (wbond) [inline repository/i.json]: You'll at least need to remove the word `Package`.

**Comment by Bernardoow** [comment]
@wbond I changed name.

**Pull request merged**